tests/conftest.py

```diff
@@ -103,8 +103,14 @@ def packages_path():
 
 def generate_package_path_fixture(package_name):
     @pytest.fixture
-    def fixture(packages_path):
-        return os.path.join(packages_path, package_name)
+    def fixture(packages_path, tmp_path):
+        package_path = os.path.join(packages_path, package_name)
+        if 'flit' in package_name:
```

Comment:
We'll end up copying all but one package this way - we only need to do it for setuptools, right?  That's only `test-bad-wheel` and `test-setuptools` AFAICT.